packages/core-entities/src/models/views/YieldViews.ts

```diff
@@ -633,10 +611,12 @@ export const YieldViews = (self: Instance<typeof NetworkModel>) => {
     getDebtOrCollateralFactor,
     getDefaultLeveragedNTokenAPYs,
     getDefaultVaultAPYs,
+    getNTokenData,
     getLeveragedNTokenData,
     getAllListedVaultsData,
-    getPortfolioStateZeroEarnData,
-    getPortfolioStateZeroBorrowData,
-    getPortfolioStateZeroLeveragedData
+    getFCashLendData,
```

Comment:
Rename these function names so they are more descriptive:
`getAllNTokenYields`, `getAllFCashYields`, etc.

"Data" does not describe accurately what is being returned.